### PR TITLE
[release-4.15] OCPBUGS-36367: update whereabouts crd

### PR DESCRIPTION
**Pull request by mlguerrero12**
Manual cherry-pick https://github.com/openshift/cluster-network-operator/pull/2419

### CODE REVIEW

File: bindata/network/additional-networks/crd/001-crd.yaml [hunk 1]
```diff
@@ -48,6 +48,9 @@ spec:
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.1
+  creationTimestamp: null
```
Comment (jcaamano) [inline bindata/network/additional-networks/crd/001-crd.yaml:53]: Surprised to see `creationTimestamp`

[the remark this answers is not in the document]
Comment (mlguerrero12) [reply bindata/network/additional-networks/crd/001-crd.yaml:53]: That is the version set in whereabouts 4.15
https://github.com/openshift/whereabouts-cni/blob/release-4.15/hack/install-kubebuilder-tools.sh

I'm generated the files using this script
https://github.com/openshift/whereabouts-cni/blob/master/hack/generate-code.sh

My goal here was to be consistent with the versions used on this release.

Comment (mlguerrero12) [reply bindata/network/additional-networks/crd/001-crd.yaml:53]: That I got by running the script in whereabouts to generate the files
https://github.com/openshift/whereabouts-cni/blob/master/hack/generate-code.sh

**Pull request merged**